renku/cli/utils/curses.py

```diff
@@ -0,0 +1,322 @@
+
+            self.help_overlay = curses.newwin(
+                self.HELP_OVERLAY_HEIGHT, self.HELP_OVERLAY_WIDTH, 0, self.cols - self.HELP_OVERLAY_WIDTH
+            )
+            curses.panel.new_panel(self.help_overlay)
+            self.help_overlay.border()
+
+            content = (
+                "Navigate using arrow keys\n"
+                "Press <enter> to show activity details\n"
+                "Press <h> to show/hide this help\n"
+                "Press <q> to exit\n"
+            )
+            self._add_multiline_text_with_wrapping(self.help_overlay, content)
+            self.help_overlay.overlay(screen)
+        else:
+            del self.help_overlay
+            self.help_overlay = None
+
+    def _move_viewscreen(self):
+        """Move viewscreen to include selected activity."""
+        if self.activity_start.x - 1 < self.x_pos:
+            self.x_pos = max(self.activity_start.x - 1, 0)
+        elif self.activity_end.x + 1 > self.x_pos + self.cols:
+            self.x_pos = min(self.activity_end.x + 1, self.content_max_x) - self.cols - 1
+
+        if self.activity_start.y - 1 < self.y_pos:
+            self.y_pos = max(self.activity_start.y - self.vertical_space - 3, 0)
+        elif self.activity_end.y + 1 > self.y_pos + self.rows:
+            self.y_pos = min(self.activity_end.y + self.vertical_space + 3, self.content_max_y) - self.rows - 1
+
+    def _loop(self, screen):
+        """The interaction loop."""
+        running = True
+
+        while running:
+            input_char = screen.getch()
+
+            # update screen size to deal with resizes
+            self.rows, self.cols = screen.getmaxyx()
+
+            # handle keypress
+            if input_char == curses.KEY_DOWN or chr(input_char) == "k":
+                self._change_layer(1)
+            elif input_char == curses.KEY_UP or chr(input_char) == "i":
+                self._change_layer(-1)
+            elif input_char == curses.KEY_RIGHT or chr(input_char) == "l":
+                self._change_layer_position(1)
+            elif input_char == curses.KEY_LEFT or chr(input_char) == "j":
+                self._change_layer_position(-1)
+            elif input_char == curses.KEY_ENTER or input_char == 10 or input_char == 13:
+                self._update_activity_overlay(screen)
+            elif chr(input_char) == "h":
+                self._update_help_overlay(screen)
+            elif input_char < 256 and chr(input_char) == "q":
+                running = False
+
+            self._unblink_text(self.content_pad, self.activity_start, self.activity_end, bold=True)
+            self._select_activity()
+            self._blink_text(self.content_pad, self.activity_start, self.activity_end, bold=True)
+
+            self._move_viewscreen()
+
+            self._refresh(screen)
+
+    def _main(self, screen):
+        """The main execution method for wrapped curses execution."""
+        self._init_curses(screen)
+        self._loop(screen)
```

Comment:
@Panaetius this patch unfortunately does not really fix the problem, as in there's no stacktrace, since it's swallowed up, but the interactive mode has no output at all